.projenrc.ts

```diff
@@ -11,7 +11,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   projenrcTs: true,
   author: 'Amazon Web Services',
   authorAddress: 'aws-cdk-dev@amazon.com',
-  cdkVersion: '2.0.0',
+  cdkVersion: '2.83.0',
```

Comment:
same comment about the cdkVersion being the minimal compatible version. Don't think we should change

Reply:
Just changed to 2.28.0 as described in the PR description.